Components/Input/input.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../../css/header.css" />
+  <link rel="stylesheet" href="../../css/sidebar.css" />
+  <link rel="stylesheet" href="../../css/content.css" />
+  <link rel="stylesheet" href="../../css/divider.css" />
+  <link rel="stylesheet" href="../../css/container.css" />
+  <link rel="stylesheet" href="../../css/media-query.css" />
+  <link rel="stylesheet" href="https://storm-components.netlify.app/components.css" />
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <title>Input | Storm UI</title>
+</head>
+
+<body>
+  <div class="container">
+    <nav class="navbar">
+      <a class="name" href="../../index.html"><i class="fas fa-bolt"></i> STORM</a>
+      <div class="icon-container">
+        <a href="https://github.com/b0llu/storm-documentation-site/tree/developer">
+          <i class="fab fa-github icon"></i>
+        </a>
+        <a href="https://twitter.com/TheBestDhruv">
+          <i class="fab fa-twitter icon"></i>
+        </a>
+        <a>
+          <i id="toggle-theme" class="fas fa-moon icon"></i>
+        </a>
+      </div>
+    </nav>
+    <div id="sidebar-menu" class="side-bar position-hidden">
```

Comment:
You should use semantic Html rather than using div everywhere like for sidebar use aside and main for main container

Reply:
thanks for pointing it out changing it asap!